.gitignore

```diff
@@ -20,4 +20,13 @@
 .rspec-failures
 
 # Ignore public uploads
-public/uploads
\ No newline at end of file
+public/uploads
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
```

Comment:
Maybe we don't need to add yarn files in gitignore if we use npm. What do you think?

Reply:
You are all right!